src/lib/viewers/image/ImageViewer.js

```diff
@@ -17,12 +18,30 @@ class ImageViewer extends ImageBaseViewer {
         this.updatePannability = this.updatePannability.bind(this);
         this.handleImageDownloadError = this.handleImageDownloadError.bind(this);
         this.handleAssetAndRepLoad = this.handleAssetAndRepLoad.bind(this);
+        this.handleZoomEvent = this.handleZoomEvent.bind(this);
+        this.getViewportDimensions = this.getViewportDimensions.bind(this);
```

Comment:
Is this binding needed if the 'handleZoomEvent' is already bound?

Reply:
Not needed